docs/cassandra/cassandra-optimization.md

```diff
@@ -0,0 +1,202 @@
+# geotrelis.cassandra.optimization
```

Comment:
@pomadchin - as per our gitter discussion, here's a summary doc. on what I did.  Suggestions welcome.

Reply:
That's great! Figuring out how to merge everything